### PR TITLE
add sobanasFunction index.js

### PR DESCRIPTION
**Pull request by sobanaq**

### CODE REVIEW

**Comment by sobanaq** [comment]
Nick please review.

File: index.js [hunk 1]
```diff
@@ -1,3 +1,7 @@
+
+const sobanasFunction = () => {
+  console.log("Hello, I'm Sobana.");
+
```
Comment (NickChew) [inline index.js:4]: missing {

**Review by NickChew — CHANGES_REQUESTED** [review]
Please check comments

File: index.js (hunk 1 above)
Comment (NickChew) [reply index.js:4]: great stuff!

**Review by NickChew — APPROVED** [review]

**Pull request merged**